include/curl/curl.h

```diff
@@ -869,6 +869,13 @@ typedef int
                           const struct curl_khkey *foundkey, /* found */
                           enum curl_khmatch, /* libcurl's view on the keys */
                           void *clientp); /* custom pointer passed from app */
+ 
+typedef int
+  (*curl_ssh_hostkeycheck_callback) (void *clientp,/* custom pointer passed from app */
+                          int keytype, /* LIBSSH2_HOSTKEY_TYPE */
+                          const char * key, /*hostkey to check*/
```

Comment:
Remove the space after the asterisk to follow our code style.

Reply:
ok, done